frontend/src/pages/connectionTypes/ConnectionTypesTableRow.tsx

```diff
@@ -0,0 +1,118 @@
+import * as React from 'react';
+import { ActionsColumn, Td, Tr } from '@patternfly/react-table';
+import {
+  Label,
+  Switch,
+  Text,
+  TextContent,
+  Timestamp,
+  TimestampTooltipVariant,
+} from '@patternfly/react-core';
+import { ConnectionTypeConfigMapObj } from '~/concepts/connectionTypes/types';
+import { relativeTime } from '~/utilities/time';
+import { updateConnectionTypeEnabled } from '~/services/connectionTypesService';
+import useNotification from '~/utilities/useNotification';
+import { TableRowTitleDescription } from '~/components/table';
+
+type ConnectionTypesTableRowProps = {
+  obj: ConnectionTypeConfigMapObj;
+  onUpdate: () => void;
+};
+
+const ConnectionTypesTableRow: React.FC<ConnectionTypesTableRowProps> = ({ obj, onUpdate }) => {
+  const notification = useNotification();
+  const [isUpdating, setIsUpdating] = React.useState(false);
+  const [isEnabled, setIsEnabled] = React.useState(false);
+  const createdDate = obj.metadata.creationTimestamp
+    ? new Date(obj.metadata.creationTimestamp)
+    : undefined;
+
+  const onUpdateEnabled = (enabled: boolean) => {
+    setIsUpdating(true);
+    setIsEnabled(enabled);
+
+    updateConnectionTypeEnabled(obj, enabled)
+      .then((response) => {
+        setIsUpdating(false);
+        if (response.success) {
+          onUpdate();
+          return;
+        }
+        notification.error(
+          `Error ${enabled} ? 'enabling' : 'disabling'} the connection type`,
+          response.error,
+        );
+        setIsEnabled(!enabled);
+      })
+      .catch((e) => {
+        notification.error(
+          `Error ${enabled} ? 'enabling' : 'disabling'} the connection type`,
+          e.message,
+        );
+        setIsEnabled(!enabled);
+      })
+      .finally(() => {
+        setIsUpdating(false);
+      });
+  };
+
+  React.useEffect(() => {
+    setIsEnabled(obj.metadata.annotations?.['opendatahub.io/enabled'] === 'true');
+  }, [obj.metadata.annotations]);
+
+  return (
+    <Tr>
+      <Td dataLabel="Name" width={70}>
+        <TableRowTitleDescription
+          title={obj.metadata.annotations?.['openshift.io/display-name'] || obj.metadata.name}
+          description={obj.metadata.annotations?.['openshift.io/description']}
+        />
+      </Td>
+      <Td dataLabel="Creator" data-testid="connection-type-creator">
+        {obj.metadata.ownerReferences?.length ? (
+          <Label data-testid="connection-type-user-label">Pre-installed</Label>
+        ) : (
+          <TextContent>
+            <Text>{obj.metadata.annotations?.['opendatahub.io/username'] || 'unknown'}</Text>
+          </TextContent>
```

Comment:
I'm confused with the expectation in ODH. PF and mocks use a standard sized font and ODH seems to make the font size smaller. By using `Text` here, it bumps the font size back up to what PF wants. But that doesn't align with the name columns now.